src/components/Card.svelte

```diff
@@ -29,8 +30,10 @@
 		return artistsOneline;
 	}
 	function toggleExpand(): void {
-		if (expand && audio) {
-			audio.pause();
+		if ($ControlStore.currentSongId === songDetails.songid) {
+			$ControlStore.currentSongId = '';
```

Comment:
nit: why is empty string better than undefined? seems like you're circumventing a sound typescript check

if ure changing it just to avoid having to check if something is undefined, then thats sus.

Reply:
Initialization will be changed to empty string so that the type is consistent with this

Reply:
updated